piker/ui/_flows.py

```diff
@@ -1208,18 +890,22 @@ def render(
         elif (
             append_length > 0
             and do_append
-            and not should_redraw
         ):
-            # print(f'{array_key} append len: {append_length}')
-            new_x = x_out[-append_length - 2:]  # slice_to_head]
-            new_y = y_out[-append_length - 2:]  # slice_to_head]
+            print(f'{array_key} append len: {append_length}')
```

Comment:
FYI this was a failed attempt at trying to get `QPainterPath` "appends" working by using graphics obj apis do do the appends at the path level instead of via a full redraw).

I'm leaving the commented code in for the future whenever we decide to take another try at it.